assets/components/yourContribution/yourContribution.jsx

```diff
@@ -7,25 +7,46 @@ import React from 'react';
 import PageSection from 'components/pageSection/pageSection';
 import PaymentAmount from 'components/paymentAmount/paymentAmount';
 import Secure from 'components/secure/secure';
+
 import { type Currency } from 'helpers/internationalisation/currency';
+import { type IsoCountry } from 'helpers/internationalisation/country';
+import { type Contrib as ContributionType } from 'helpers/contributions';
 
 
 // ----- Types ----- //
 
 type PropTypes = {
-  contributionType: string,
+  contributionType: ContributionType,
+  country: IsoCountry,
   amount: number,
   currency: Currency,
 };
 
 
+// ----- Setup ----- //
+
+function getHeading(cT: ContributionType, c: IsoCountry): string {
```

Comment:
I would change `c` to be `country` and `cT` to be `countributionType` since are more meaningful names and therefore avoid to jump back to the method's signature.